source/reference/minio-server/minio-server.rst

```diff
@@ -522,6 +540,16 @@ refers to the specific storage tier on which to store a given object.
    This environment variable only applies to deployments with
    :ref:`Erasure Coding <minio-erasure-coding>` enabled.
 
+   The minimum value at startup is ``0``.
+   0 parity setups have no erasure coding protections and rely entirely on the storage controller or resource for availability / resiliency. 
+
+   The maximum value is 1/2 the erasure set stripe size.
+   For example, a deployment with erasure set stripe size of 16 has a maximum standard parity of 8.
+
+   You can change the Standard parity after startup to a value between ``1`` and ``1/2(ERASURE_SET_SIZE)``.
```

Comment:
Can use the `math` role here:

```suggestion
   You can change the Standard parity after startup to a value between ``1`` and :math:`\tfrac{1}{2}\ (ERASURE_SET_SIZE)`.
```

https://www.sphinx-doc.org/en/master/usage/restructuredtext/roles.html#role-math
https://ctan.mirrors.hoobly.com/info/short-math-guide/short-math-guide.pdf

Reply:
Thank you for putting the effort into this so I dont have to